src/openrct2-ui/interface/LandTool.cpp

```diff
@@ -65,7 +65,7 @@ void LandTool::ShowSurfaceStyleDropdown(rct_window* w, rct_widget* widget, uint8
     for (size_t i = 0; i < MAX_TERRAIN_SURFACE_OBJECTS; i++)
     {
         const auto surfaceObj = static_cast<TerrainSurfaceObject*>(objManager.GetLoadedObject(ObjectType::TerrainSurface, i));
-        if (surfaceObj != nullptr)
+        if (surfaceObj != nullptr && surfaceObj->NumImagesLoaded > 1)
```

Comment:
Why is this requried? I'm sure its fine and makes sense just want to know the reason.

Reply:
If the user does not have RCT1, we can't load any sprites for the surfaces in question (except for the preview image). This code hides such surfaces from the dropdown.

Reply:
```suggestion
        // NumImagesLoaded can be 1 for RCT1 surfaces if the user does not have RCT1 linked.
        if (surfaceObj != nullptr && surfaceObj->NumImagesLoaded > 1)
```